faq/templates/faq/inclusion_tags/faq_listing_item.html

```diff
@@ -0,0 +1,8 @@
+{% load ultracache_tags listing_tags i18n %}
```

Comment:
Almost the same, except I think it's preferable that the `object.question` be used instead of `object.title`